### PR TITLE
Update/lint the Transfer Learning with YAMNet tutorial, remove unused TensorFlow Datasets import

### PR DESCRIPTION
**Pull request by 8bitmp3**

### CODE REVIEW

File: site/en/tutorials/audio/transfer_learning_audio.ipynb [hunk 1]
```diff
@@ -131,27 +130,25 @@
       "source": [
         "## About YAMNet\n",
         "\n",
-        "YAMNet is an audio event classifier that takes audio waveform as input and makes independent predictions for each of 521 audio events from the [AudioSet](https://research.google.com/audioset/) ontology.\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/audio/transfer_learning_audio.ipynb:134]: It's important not to loose the "independent predictions for each of 521 ..." here.

I talkes to the authors., The model really does do 512 independant binary classifications. Not a 512-way softmax. That distinction is important to correctly interpret the results.

File: site/en/tutorials/audio/transfer_learning_audio.ipynb [hunk 2]
```diff
@@ -131,27 +130,25 @@
...
         "\n",
-        "One specific use of YAMNet is as a high-level feature extractor: the `1024-D` embedding output of YAMNet can be used as the input features of another shallow model which can then be trained on a small amount of data for a particular task. This allows the quick creation of specialized audio classifiers without requiring a lot of labeled data and without having to train a large model end-to-end.\n",
+        "One specific use of YAMNet is as a high-level feature extractor - the 1,024-dimensional embedding output. You will use the base (YAMNet) model's input features and feed them into your shallower model consisting of one hidden [Dense](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense) layer. Then, you will train the network on a small amount of data for audio classification _without_ requiring a lot of labeled data and training end-to-end. (This is similar to [transfer learning for image classification with TensorFlow Hub](https://www.tensorflow.org/tutorials/images/transfer_learning_with_hub) for more information.)\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/audio/transfer_learning_audio.ipynb:141]: `layers.Dense` will auto-link on the site.

File: site/en/tutorials/audio/transfer_learning_audio.ipynb [hunk 3]
```diff
@@ -197,9 +194,9 @@
         "id": "mBm9y9iV2U_-"
       },
       "source": [
-        "You will need a function to load the audio files. They will also be used later when working with the training data.\n",
+        "You will need a function to load audio files, which will also be used later when working with the training data. (Learn more about rading audio files and their labels in [Simple audio recognition](https://www.tensorflow.org/tutorials/audio/simple_audio#reading_audio_files_and_their_labels).\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/audio/transfer_learning_audio.ipynb:197]: typo

File: site/en/tutorials/audio/transfer_learning_audio.ipynb [hunk 4]
```diff
@@ -345,7 +342,7 @@
         "\n",
         "and all the audio files are in `./datasets/ESC-50-master/audio/`\n",
         "\n",
-        "You will create a pandas dataframe with the mapping and use that to have a clearer view of the data.\n"
+        "You will create a pandas DataFrame with the mapping and use that to have a clearer view of the data.\n"
```
Comment (MarkDaoust) [inline site/en/tutorials/audio/transfer_learning_audio.ipynb:345]: backticks for code. `DataFrame`

File: site/en/tutorials/audio/transfer_learning_audio.ipynb [hunk 5]
```diff
@@ -371,11 +368,11 @@
       "source": [
         "### Filter the data\n",
         "\n",
-        "Given the data on the dataframe, you will apply some transformations:\n",
+        "Now that the data is tored in the DataFrame, apply some transformations:\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/audio/transfer_learning_audio.ipynb:371]: typo

**Review by MarkDaoust — CHANGES_REQUESTED** [review]

File: site/en/tutorials/audio/transfer_learning_audio.ipynb (hunk 4 above)
Comment (8bitmp3) [reply site/en/tutorials/audio/transfer_learning_audio.ipynb:345]: That's a good point @MarkDaoust . Will add them here. 

Can we use both like pandas does? https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe

File: site/en/tutorials/audio/transfer_learning_audio.ipynb (hunk 5 above)
Comment (8bitmp3) [reply site/en/tutorials/audio/transfer_learning_audio.ipynb:371]: When Thor sees a DataFrame

Thor

File: site/en/tutorials/audio/transfer_learning_audio.ipynb (hunk 1 above)
Comment (8bitmp3) [reply site/en/tutorials/audio/transfer_learning_audio.ipynb:134]: Done.

```
[YAMNet](https://github.com/tensorflow/models/tree/master/research/audioset/yamnet) 
is a pre-trained neural network that employs the [MobileNetV1](https://arxiv.org/abs/1704.04861)
depthwise-separable convolution architecture. It can use an audio waveform as input and make
independent predictions for each of the 521 audio events from the [AudioSet](http://g.co/audioset) corpus.
```

**Review by lamberta — APPROVED** [review]
